README.md

```diff
@@ -119,13 +119,21 @@ The command above mounts both of these directories.
 ## Running as an executable
 Supported only on x86_64!
 
-Run the following to have gprofiler running continuously, uploading to Granulate Performance Studio:
+First, check if gProfiler is already running - run `pgrep gprofiler`. You should see 3 PIDs (staticx bootloader, PyInstaller and gProfiler's Python itself). If no processes match, then gProfiler is not running.
+
+Run the following to have gprofiler running continuously, in the background, uploading to Granulate Performance Studio:
 ```bash
 wget https://github.com/Granulate/gprofiler/releases/latest/download/gprofiler
 sudo chmod +x gprofiler
-sudo ./gprofiler -cu --token <token> --service-name <service> [options]
+sudo sh -c "setsid ./gprofiler -cu --token <token> --service-name <service> [options] > /dev/null 2>&1 &"
```

Comment:
```suggestion
sudo sh -c "setsid ./gprofiler -cu --token <token> --service-name <service> [options] > /dev/null 2>&1 &"
pgrep gprofiler  # Make sure gProfiler is running
```
I actually meant that the `pgrep` command to be here as well, as some people (like me sometimes) only read the commands and skip the docs

Reply:
If so, we should add a `sleep` (as it takes a while for staticx & PyInstaller to start, if both commands are pasted together it's not enough time for it to start)

Reply:
Sounds good to me